manifests/params.pp

```diff
@@ -13,6 +13,7 @@
   case $::operatingsystem {
     'CentOS',
     'RedHat',
+    'XCP',   
```

Comment:
please remove trailing whitespaces
Also check that travis-ci reports OK

Reply:
addet another commit and removed the whitespaces